blog/index.html

```diff
@@ -95,6 +98,55 @@ <h3>
               {% endfor %}
           {% endif %}
     </p>
+    </div>
+  <div class="col-sm-3">
+    <img class="card-img" src="{{post.thumbnail | relative_url}}" style="object-fit: cover; height: 90%" alt="image">
+  </div>
```

Comment:
please put only this part should be inside `{% if post.thumbnail %} {% endif %}` so that the rest of the code is shared and not duplicated

Reply:
Agreed and done.

Reply:
Actually it was needed in two parts, to open and close the divs.